specs/lightclient/beacon-chain.md

```diff
@@ -151,13 +172,13 @@ def process_block(state: BeaconState, block: BeaconBlock) -> None:
 ```python
 def process_sync_committee(state: BeaconState, body: BeaconBlockBody) -> None:
     # Verify sync committee aggregate signature signing over the previous slot block root
-    previous_slot = max(state.slot, Slot(1)) - Slot(1)
+    previous_slot = max(Slot(state.slot), Slot(1)) - Slot(1)
```

Comment:
Just curious, why the Slot wrapper around `state.slot`? Isn't `state.slot` of type Slot already?

Reply:
It's a side effect of the new OOP SSZ (inheritance) implementation: https://github.com/ethereum/eth2.0-specs/pull/2141#discussion_r533627458

Statically, Mypy doesn't recognize the parent class fields when checking the child class object.